composer.json

```diff
@@ -11,23 +11,25 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "^2.6|^3.0",
-        "symfony/security-http": "^2.6|^3.0",
-        "mixpanel/mixpanel-php": "~2.6"
+        "php": "^7.1.3",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/expression-language": "^4.3 || ^5.0",
+        "symfony/security-http": "^4.3 || ^5.0",
+        "mixpanel/mixpanel-php": "~2.8"
     },
     "require-dev": {
-        "phpspec/phpspec": "^3.0"
+        "phpspec/phpspec": "^5.0 || ^6.0",
```

Comment:
Why using two different version? Use the latest 6.x which is compatible with Symfony 4 and 5.